deps/build.jl

```diff
@@ -11,7 +11,7 @@ ignore_system_font_libs(n, h) = !(haskey(ENV, "JULIA_BINDEPS_IGNORE_SYSTEM_FONT_
 
 deps = [
     libffi = library_dependency("ffi", aliases = ["libffi"], group = group1)
-    gobject = library_dependency("gobject", aliases = ["libgobject-2.0-0", "libgobject-2.0", "libgobject-2_0-0", "libgobject-2.0.so.0"], depends=[libffi], group = group1)
+    gobject = library_dependency("gobject", aliases = ["libgobject-2.0.so.0", "libgobject-2.0-0", "libgobject-2.0", "libgobject-2_0-0"], depends=[libffi], group = group1)
```

Comment:
The commit message is "put more specific gobject alias first". This ensures that the more specific pattern is tried first. In this case it makes a difference because the build box has a symlink "libgobject-2.0.so -> libgobject-2.0.so.0" but the deployment box does not  (it only has "libgobject-2.0.so.0"). Putting "libgobject-2.0.so.0" first ensures that this is what ends up in the generated code.